libs/utils/trace.py

```diff
@@ -171,14 +163,9 @@ def setXTimeRange(self, t_min=None, t_max=None):
         :param t_max: upper bound
         :type t_max: int or float
         """
-        if t_min is None:
-            self.x_min = 0
-        else:
-            self.x_min = t_min
-        if t_max is None:
-            self.x_max = self.time_range
-        else:
-            self.x_max = t_max
+        self.x_min = t_min if t_min is not None else self.start_time
+        self.x_max = t_max if t_max is not None else self.start_time + self.time_range
```

Comment:
Added a None check here for the sake of symmetry. A 0 x_max doesn't make a lot of sense, but I think it's less confusing to let that pass than to force the x_max to its maximum possible value if 0 is given.